code/clue.js

```diff
@@ -41,19 +189,73 @@ const randomSelector = array => {
 // With a killer, a weapon and a room.
 // The values will be set later.
 
+let mystery = {
+  killer:'',
+  weapon:'',
+  room:''
+}
+
+
+
+
 // This function will be invoked when you click on the killer card.
+
 const pickKiller = () => {
   // This will randomly select a killer from the suspects. And add that to the mystery object.
   mystery.killer = randomSelector(suspects)
+  
+  // Shuffle weapon function
+  shuffleFavouriteWeapon = () => {
+    mystery.killer.favouriteWeapon = randomSelector(weapons); 
+  };
```

Comment:
You are using semicolons here but everywhere else you are not using semicolon. Be consistent.

Reply:
Once again, good job Jessica!
One more thing to point out - in line 208 there is a declaration of variable without any keyword `let` or `const`. Because of that you've created global variable `shuffleFavouriteWeapon`. Briefly speaking, of you would create `const shuffleFavouriteWeapon = ...`, it would be scoped to in `pickKiller` function only and unavailable from outer scope. Because you omitted `const`, your variable somehow "slipped out" from `pickKiller` function.
This is huge antipattern, we should never (well, in 99.99% cases being honest) create global variables. Please, fix that asap

Reply:
Thank you for that Maks! I added const to the function. I also moved the function out of pickKiller. I don't know if that was what you wanted but perhaps makes the function more useful?

Reply:
Probably it is, good idea :)